enable-tls-for-mysql-client: add info about rotation of X.509 TLS certs (#2946)
pingcap/docs-tidb-operator: en/enable-tls-for-mysql-client.md

kubectl get secret -n ${namespace} ${cluster_name}-tidb-client-secret  -ojsonpath='{.data.tls\.key}' | base64 --decode > client-tls.key
kubectl get secret -n ${namespace} ${cluster_name}-tidb-client-secret  -ojsonpath='{.data.ca\.crt}'  | base64 --decode > client-ca.crt
```

``` shell
mysql --comments -uroot -p -P 4000 -h ${tidb_host} --ssl-cert=client-tls.crt --ssl-key=client-tls.key --ssl-ca=client-ca.crt
```

Finally, to verify whether TLS is successfully enabled, refer to [checking the current connection](https://docs.pingcap.com/tidb/stable/enable-tls-between-clients-and-servers#check-whether-the-current-connection-uses-encryption).

## Reload certificates

- If you generate the certificate and key files manually using `cfssl`, you must update the corresponding Secret manually.
- If you generate the certificate and key files using `cert-manager`, the Secret is updated automatically whenever a new certificate is issued.

To let TiDB use the new certificate, run [`ALTER INSTANCE RELOAD TLS`](https://docs.pingcap.com/tidb/stable/sql-statement-alter-instance/#reload-tls).

> You can run the following SQL statement to check the `Ssl_server_not_before` and `Ssl_server_not_after` status variables to check the validity period on the certicates.

Reviewer: ```suggestion
To verify the certificate validity period, run the following SQL statement to check the `Ssl_server_not_before` and `Ssl_server_not_after` status variables:
```
